README.md

```diff
@@ -144,13 +144,13 @@ The file `secrets.yml` will tell `summon` which keys to get from `gopass`.
 | ----------------------------- | -------------------------------------------- |
 | `npm run start`               | Compiles and hot-reloads for development. Will serve the project under `http://localhost:8080` (or the next available port if `8080` is already used, see console output). |
 | `npm run build`               | Compiles all file without bundling and minification |
+| `npm run build:(dev|int|prod)`| Compiles all file for the according `mode` |
```

Comment:
Somehow this is not rendered correctly:
![image](https://user-images.githubusercontent.com/67745584/208917005-38914186-6309-42f1-8c0b-e6b7b4728f79.png)